packages/app/src/elements/item-view.ts

```diff
@@ -234,24 +237,24 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
-    private _removeExpiration(skipSaving = false) {
+    private _removeExpiration(skipSaving = this._editing) {
```

Comment:
We don't want to leave edit mode or save if they add/remove an expiration date while in edit mode.